core/esmf-aspect-model-validator/src/main/java/org/eclipse/esmf/aspectmodel/shacl/RustLikeFormatter.java

```diff
@@ -231,6 +231,12 @@ private boolean formatNode( final RDFNode node ) {
          if ( node.asNode().equals( NodeConst.nodeRDFType ) ) {
             spacedIfPossible( "a " );
```

Comment:
Is a space behind 'a' intended?

Reply:
In this case yes, because there will always be a space after the 'a ' (= rdf:type) token when formatting, because it can only appear as the predicate in a triple and is followed by some object.